lua/cfc_voice_channels/core/sv/sv_channel_control.lua

```diff
@@ -97,6 +98,18 @@ function cfc_voice:joinChannel(ply, channel)
     table.insert(channel.Users, ply)
 end
 
+function cfc_voice:ChannelPlayerDisconnect(channel)
```

Comment:
Can this be renamed to `function cfc_voice:OnChannelPlayerDisconnect(channel)` ?